docs/guides/getting-started-orchestrate-microservices.md

```diff
@@ -72,5 +72,5 @@ Congratulations! You successfully built your first microservice orchestration so
 
 ## Additional resources and next steps
 
-- Learn more about Camunda Platform 8 and what it can do by reading [What is Camunda Platform 8](/components/concepts/what-is-camunda-platform-8.md) or watching our [Platform Overview video](https://bit.ly/3TjNEm7) in Camunda Academy.
+- Learn more about Camunda Platform 8 and what it can do by reading [What is Camunda Platform 8](/components/concepts/what-is-camunda-8.md) or watching our [Platform Overview video](https://bit.ly/3TjNEm7) in Camunda Academy.
```

Comment:
```suggestion
- Learn more about Camunda 8 and what it can do by reading [What is Camunda 8](/components/concepts/what-is-camunda-8.md) or watching our [Platform Overview video](https://bit.ly/3TjNEm7) in Camunda Academy.
```

Reply:
Looks like there are other "Platforms" in this file too.

Reply:
Same here 👍